requirements.txt

```diff
@@ -1,5 +1,6 @@
 numpy~=1.19.5
 packaging~=20.8
+pymongo<=3.12.0
```

Comment:
Is this still needed after the changes you made?

Reply:
only if you want to be extra careful. If I successfully found all the breaking changes then its no longer needed.

Reply:
based on strax' tests I think it's indeed good to add this requirement